config/config.js

```diff
@@ -4,179 +4,180 @@
  * Module dependencies.
  */
 var _ = require('lodash'),
-    chalk = require('chalk'),
-    glob = require('glob'),
-    fs = require('fs'),
-    path = require('path');
+  chalk = require('chalk'),
+  glob = require('glob'),
+  fs = require('fs'),
+  path = require('path');
 
 /**
  * Get files by glob patterns
  */
-var getGlobbedPaths = function(globPatterns, excludes) {
-    // URL paths regex
-    var urlRegex = new RegExp('^(?:[a-z]+:)?\/\/', 'i');
-
-    // The output array
-    var output = [];
-
-    // If glob pattern is array so we use each pattern in a recursive way, otherwise we use glob
-    if (_.isArray(globPatterns)) {
-        globPatterns.forEach(function(globPattern) {
-            output = _.union(output, getGlobbedPaths(globPattern, excludes));
-        });
-    } else if (_.isString(globPatterns)) {
-        if (urlRegex.test(globPatterns)) {
-            output.push(globPatterns);
-        } else {
-            var files = glob.sync(globPatterns);
-            if (excludes) {
-                files = files.map(function(file) {
-                    if (_.isArray(excludes)) {
-                        for (var i in excludes) {
-                            file = file.replace(excludes[i], '');
-                        }
-                    } else {
-                        file = file.replace(excludes, '');
-                    }
-                    return file;
-                });
+var getGlobbedPaths = function (globPatterns, excludes) {
+  // URL paths regex
+  var urlRegex = new RegExp('^(?:[a-z]+:)?\/\/', 'i');
+
+  // The output array
+  var output = [];
+
+  // If glob pattern is array so we use each pattern in a recursive way, otherwise we use glob
+  if (_.isArray(globPatterns)) {
+    globPatterns.forEach(function (globPattern) {
+      output = _.union(output, getGlobbedPaths(globPattern, excludes));
+    });
+  } else if (_.isString(globPatterns)) {
+    if (urlRegex.test(globPatterns)) {
+      output.push(globPatterns);
+    } else {
+      var files = glob.sync(globPatterns);
+      if (excludes) {
+        files = files.map(function (file) {
+          if (_.isArray(excludes)) {
+            for (var i in excludes) {
+              file = file.replace(excludes[i], '');
```

Comment:
what's up with this odd diff here? looks like just spacing, but nothing was actually changed in the code, right?